copilot/fsd-account-store/manifest.yml

```diff
@@ -0,0 +1,55 @@
+# The manifest for the "account-store" service.
+# Read the full specification for the "Backend Service" type at:
+#  https://aws.github.io/copilot-cli/docs/manifest/backend-service/
+
+# Your service name will be used in naming your resources like log groups, ECS services, etc.
+name: fsd-account-store
+type: Backend Service
+
+# Distribute traffic to your service.
+http:
+  # Requests to this path will be forwarded to your service.
+  # To match all requests you can use the "/" path.
+  path: '/'
+  # You can specify a custom health check path. The default is "/".
+  healthcheck: '/healthcheck'
+
+# Configuration for your containers and service.
+image:
+  # Docker build arguments. For additional overrides: https://aws.github.io/copilot-cli/docs/manifest/lb-web-service/#image-build
+  location: ghcr.io/communitiesuk/funding-service-design-account-store:latest
+  # Port exposed through your container to route traffic to it.
+  port: 8080
+
+# Valid values: https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-cpu-memory-error.html
+# Number of CPU units for the task.
+cpu: 256
+# Amount of memory in MiB used by the task.
+memory: 512
+
+# See https://aws.github.io/copilot-cli/docs/manifest/lb-web-service/#platform
+platform: linux/x86_64
+
+# Number of tasks that should be running in your service.
+count: 1
+
+# Enable running commands in your container.
+exec: true
+
+network:
+  connect: true # Enable Service Connect for intra-environment traffic between services.
+
+# Optional fields for more advanced use-cases.
+#
+# Pass environment variables as key value pairs.
+variables:
+  # SENTRY_DSN: "https://8092c54bdc3a4b4ba124dea42b836504@o1432034.ingest.sentry.io/4503918903820288"
```

Comment:
If we genuinely don't need this might as well remove?

Reply:
Oh, it's needed. Sentry is our reporting and logging mechanism. However, until we have a (mostly) linked up fully operational dev environment, I don't want to be hooking them up to prevent spam